src/refactor_nrepl/find/util.clj

```diff
@@ -20,3 +20,12 @@
              (core/suffix name)))))
   ([file occ]
    (spurious? (assoc occ :file file))))
+
+(defn divide-by
```

Comment:
How about `chunk`?  With a docstring of "Split `coll` into `n` chunks.  The final chunk might be smaller than the rest."

Reply:
Can do!

However:

imagine you see `(chunk coll 3)` in the wild, without further context. Would you imagine/remember it _divides_ the members by 3, rather than returning chunks _of size_ 3 (which is different)?

Reply:
> Would you imagine/remember it divides the members by 3, rather than returning chunks of size 3 (which is different)?

This part really confuses me.  I thought maybe I had misunderstood what `divide-by` did, but one of the tests:

```clj
(chunk [1 1 1] 3)
;;=> '((1) (1) (1))
```
Is exactly how I'd expect chunking to work: it creates 3 equally sized chunks of the input seq.

I agree that the name isn't perfect (could work exactly like `partition`), I just thought `divide-by` gave little indication of what went on.  Perhaps we can come up with something else entirely?  `partition-evenly`?  I mean it's just like partition except in the degenerate case where you get one-element partitions instead of a single partition with everything.

Reply:
> Is exactly how I'd expect chunking to work: it creates 3 equally sized chunks of the input seq.

Not sure I agree here. clojure.core chunks have a different meaning, which influences my thought.

Perhaps `in-groups-of` would do? Name picked after https://apidock.com/rails/Array/in_groups_of which does the same thing

Reply:
I like that! 👍

Reply:
Argh, I lied:

```
pry> [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12].in_groups_of(3)
=> [[1, 2, 3], [4, 5, 6], [7, 8, 9], [10, 11, 12]]
```

`in_groups_of` is exactly like `partition`, with no 'division' semantics. Not good to use the identical vocabulary for different things (which is my objection to `chunk` as well)

Reply:
I tried to look for prior art:  This function exists in flatland as [slice](https://cljdoc.org/d/org.flatland/useful/0.11.6/api/flatland.useful.seq#slice).

I'm not excited by that name, but I think it's better than `divide-by`.  FWIW I would've inlined this function a long time ago after failing to find a good name :p